src/gesture/device-type.h

```diff
@@ -26,4 +26,25 @@ enum class DeviceType {
   TOUCHSCREEN = 2,
```

Comment:
It is unlikely that we add a new device type, but it is worth to add a comment so we don't forget to update the `deviceTypeToStr` and `deviceTypeFromStr` functions.

Something like we have in `animation-type.h`:

```
// Adding a new device type? Don't forget to add it in deviceTypeToStr and
// deviceTypeFromStr
```